Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 5.9
```

Comment:
What api's do you use that swift 5.9 is required?

Reply:
Reverted in https://github.com/MountebankSwift/MountebankSwift/pull/31/commits/ba3cbde3b484a7a2f7d73f362209ca55bea61f6f